README.md

```diff
@@ -1,70 +1,157 @@
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm run build`
+This project was built using these technologies.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+-Node.js
+-React
+-Redux
+-React-Router
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Tech Stack <a name="tech-stack"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://reacttraining.com/react-router/">React Router</a></li>
+  </ul>
+</details>
 
-### `npm run eject`
+### Key Features <a name="key-features"></a>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- **Dynamic Metric Filtering**: Users can filter cryptocurrency metrics based on parameters like Id.
+- **Detailed View**: Users can view detailed information about specific cryptocurrencies.
+- **Stylish UI**: A visually appealing and user-friendly interface designed for seamless interaction.
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- [Live Demo Link](https://splendid-churros-4f2e1c.netlify.app/)
 
-## Learn More
+## Video Link <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+--sorry the video is not too good, I had a very rough week and personal stuff caught up with me I am fixing my project on a bus and I have less power and unstable internert, so kindly help me out. I will be very grateful. Thank you.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+https://drive.google.com/file/d/11x27LklaCr0u3tPisBC1Tbj1Evv5V30b/view?usp=sharing
```

Comment:
Nice job updating the video. However, as requested by the previous reviewer, the video should highlight a piece of code you find interesting or proud of. Kindly update your video to include this.

Reply:
Sir I am on a bus and I can't manage right now, my 0c is 20percent kindly be lenient